api/src/pcapi/core/mails/transactional/users/ubble/ubble_ko_reminder.py

```diff
@@ -68,24 +85,16 @@ def _find_users_that_failed_ubble_check(
             fraud_models.BeneficiaryFraudCheck.dateCreated
             >= datetime.datetime.utcnow() - relativedelta(days=days_ago + 1),
         )
+        .order_by(users_models.User.id, fraud_models.BeneficiaryFraudCheck.dateCreated)
+        .with_entities(users_models.User, fraud_models.BeneficiaryFraudCheck.reasonCodes)
         .all()
     )
 
     return [
-        (user, _get_latest_failed_fraud_check(user))
-        for user in users
+        (user, reason_codes)
+        for user, reason_codes in users
         if users_api.is_eligible_for_beneficiary_upgrade(user, user.eligibility)
         and subscription_api.get_identity_check_subscription_status(user, user.eligibility)
         == subscription_models.SubscriptionItemStatus.TODO
+        and len(set(reason_codes) & set(error_codes)) > 0
```

Comment:
(détail / style / question de goût) Je pense qu'on peut se passer du `len()` : 

```suggestion
        and set(reason_codes) & set(error_codes)
```

... ou bien, en plus verbeux : 

```suggestion
        and set(reason_codes).intersection(set(error_codes))
```